Snippets/ASB/ASB_6/Usage.cs

```diff
@@ -1,23 +1,33 @@
-﻿using NServiceBus;
+﻿using System.Configuration;
+using NServiceBus;
 
 class Usage
 {
     Usage(BusConfiguration busConfiguration)
     {
         #region AzureServiceBusTransportWithAzure
 
-        busConfiguration.UseTransport<AzureServiceBusTransport>();
+        var transport = busConfiguration.UseTransport<AzureServiceBusTransport>();
+        transport.ConnectionString("Endpoint=sb://{namespace}.servicebus.windows.net/;SharedAccessKeyName={keyname};SharedAccessKey={keyvalue}");
 
         #endregion
     }
 
-    #region AzureServiceBusTransportWithAzureHost
+    #region AzureServiceBusQueueConfigSection
```

Comment:
This has to do with the azure configuration source configuration model, if it is available as a section it could be set both from app.config as well as the .cscfg file using our naming convention